packages/react-native/ReactCommon/react/performance/timeline/PerformanceEntryCircularBuffer.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#pragma once
+
+#include "PerformanceEntryBuffer.h"
+#include "BoundedConsumableBuffer.h"
+
+namespace facebook::react {
+
+struct PerformanceEntryCircularBuffer : public PerformanceEntryBuffer {
```

Comment:
I'd use a `class` here instead of a `struct` for 2 reasons: it's extending something that's already a class, and it has enough behavior to model it as a class on its own.